src/spetlr/cosmos/cosmos_handle.py

```diff
@@ -67,3 +67,8 @@ def upsert(self, df: DataFrame, join_cols: List[str]) -> Union[DataFrame, None]:
 
     def get_tablename(self) -> str:
         return self._name
+
+    def delete_data(
+        self, comparison_col: str, comparison_limit: Any, comparison_operator: str
```

Comment:
I had a hard time making this decision as well. I considered writing an expansive Union type, but thought it was a waste of time and it would be a pain if I forgot a type that would be needed later.